class.jetpack.php

```diff
@@ -3757,7 +3757,7 @@ static function build_stats_url( $args ) {
 		 */
 		$base_url = apply_filters(
 			'jetpack_stats_base_url',
-			set_url_scheme( 'http://pixel.wp.com/g.gif' )
+			set_url_scheme( 'https://pixel.wp.com/g.gif' )
```

Comment:
If we want to use `HTTPS` all the time, we'll need to get rid of `set_url_scheme()` here.